src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -55,7 +55,7 @@ export default class ClientMetricReport {
   };
 
   packetLossPercent = (sourceMetricName?: string, ssrc?: number): number => {
-    const metricReport = ssrc ? this.streamMetricReports[ssrc] : this.globalMetricReport;
+    const metricReport = this.streamMetricReports[ssrc];
```

Comment:
This is only for `streamMetricReport`, and the metric does not exist in `globalMetricReport`, so removed the code to reduce confusion.